app/services/unpublishing_service.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class DocumentUnpublishingService
+class UnpublishingService
```

Comment:
This one should probably be 'UnpublishService' for consistency.

Reply:
Oops missed a few letters here, well spotted!